### PR TITLE
Basic beer search with OpenBeerDatabase API

### PR DESCRIPTION
**Pull request by azac**
**What does your Instant Answer do?**

Show basic beer info 

**What problem does your Instant Answer solve (Why is it better than organic links)?**

Short, encyclopedic information about beers

**What is the data source for your Instant Answer? (Provide a link if possible)**

http://openbeerdatabase.com/

**Why did you choose this data source?**

free, public domain, no rate limits

**Are there any other alternative (better) data sources?**

BreweryDB, but it's rate-limited for free users.

**What are some example queries that trigger this Instant Answer?**

beer leffe dark

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

beer lovers :)

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

screen shot 2015-03-12 at 22 29 16

## Checklist
Please place an 'X' where appropriate.

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [X] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Mac OSX
        - [X] Google Chrome
        - [X] Firefox
        - [X] Safari

    - iOS (iPhone)
        - [X] Safari Mobile
```

https://duck.co/ia/view/beer

### CODE REVIEW

**Comment by zekiel** [comment]
I'm excited for this :) 

@azac  where did you find out about DuckDuckHack? Also, did you not answer that question because this isn't your first IA?

**Comment by azac** [comment]
Hi,

@zekiel no, that's my first IA, but I simply cannot remember where I found out about DDH. I knew for a while that it's possible to create DDG search plugins, but cannot really tell where I read about it.

**Comment by moollaza** [comment]
@azac thanks a lot for this. I noticed that you've chosen OpenBeerDatabase. I would suggest you switch this to use BreweryDB as their API seems to provide much more information and also has much better documentation. As well, they seem to offer a lot more data such as Beer styles and even Events.

One of our community members had outlined a plan for several Beer Related Instant Answers, which we should try to follow and implement: https://duck.co/ideas/idea/4669/the-world-of-craft-beer

File: share/spice/beer/beer.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        Spice.add({
+            id: "beer",
+            name: "Beer",
+            data: api_result.beers[0],
+            meta: {
+                sourceName: "Open Beer Database",
+                sourceUrl: 'http://openbeerdatabase.com/'
+            },
+            templates: {
+                group: 'base',
```
Comment (moollaza) [inline share/spice/beer/beer.js]: We should be able to use the [Info](https://duck.co/duckduckhack/spice_templates_overview#info-template-group) template here, or even the Text template if we don't have much info. Seems like BreweryDB will give use lots of info that would could put into the [InfoBox](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/tvmaze/show/tvmaze_show.js#L23)

**Comment by moollaza** [comment]
Ping! @azac ares you still interested in working on this? We'd love to see it go live!

**Comment by moollaza** [comment]
@azac I'm closing this PR for now. If you're still interested in finishing, please let me know and I'll gladly re-open it for you!

**Pull request closed without merging**